submission-scripts/script-templates/magpie-alluxio

```diff
@@ -0,0 +1,143 @@
+############################################################################
+# Alluxio Core Configurations
+############################################################################
+
+# Should Alluxio be run
+#
+# Specify yes or no. Defaults to no.
+#
+export ALLUXIO_SETUP=no
+
+# Alluxio Version
+#
+export ALLUXIO_VERSION="ALLUXIOVERSIONDEFAULT"
+
+# Path to your Alluxio build/binaries
+#
+# This should be accessible on all nodes in your allocation. Typically
+# this is in an NFS mount.
+#
+export ALLUXIO_HOME="ALLUXIODIRPREFIX/alluxio-${ALLUXIO_VERSION}"
+
+# ALLUXIO_WORKER_ON_MASTER
+#
+# Most big data projects do not have a "slave" or "worker" that also
+# runs on the master node (such as Hadoop, Hbase, etc.). Alluxio is
+# an exception in which it often likes to have a worker running on the
+# master node for convenience of running tests and such.
+#
+# It is not required to run a worker on the master, but it can be
+# convenient. Specify the below to "yes" to run a worker on the
+# master.
+#
+export ALLUXIO_WORKER_ON_MASTER="yes"
+
+# Path to store data local to each cluster node, typically something
+# in /tmp. This will store local conf files and log files for your
+# job. If local scratch space is not available, consider using the
+# MAGPIE_NO_LOCAL_DIR_DIR option. See README for more details.
+#
+export ALLUXIO_LOCAL_DIR="LOCALDIRPREFIX/alluxio"
+
+# Path to store Alluxio Under File System data
+#
+# This should point to the path where you store your data
```

Comment:
per comment about UNDER_FS_DIR pointing to Lustre by default, may need to update this documentation